setup.py

```diff
@@ -38,13 +38,14 @@
         'future>0.14.0',
         'lzstring',
         'jinja2>=2.9',
-        'matplotlib>=2.1.1,<3.0.0',
+        'matplotlib>=2.1.1' + (',<3.0.0' if sys.version_info[0] == 2 else ''),  # pin for py2
```

Comment:
Looks like mpl 3.1 needs 3.6 or higher now, so this is now failing on 3.5 🤦‍♂

Reply:
Of course we can't get away with this easily. Let's see if it works now.